src/App.jsx

```diff
@@ -1,40 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
+export const App = () => {
+  const [selectedTabId, setSelectedTabid] = useState(tabs[0].id);
+  const isSelectedTab = tabs.find(tab => tab.id === selectedTabId);
```

Comment:
`find` returns you item from collection or `undefined`, so when we call variable as `is...` for example `isSelected`, `isDisabled`, `isReversed` etc. this variable should be boolean, so here you can call it just `selectedTab`, because it returns tab